README.md

```diff
@@ -56,9 +56,16 @@ Generate an admin user:
 
     bundle exec rake db:devseed    
 
-Fire up a server and import some data.
+Fire up a server:
 
     foreman start -f Procfile.dev
+    
+Or, if you already have Elasticsearch and Redis running because the web and sidekiq actions timeout in `Procfile.dev`, you can start foreman like this:
+
+    foreman start -f Procfile_no_es_redis.dev
```

Comment:
I don't think there's any getting round the fact that during initialization Rails has to query ES in order to know which model classes to create. I don't see any problem with that being the case. I'd rather we just have one Procfile for development if possible, so I suggest moving Procfile_no_es_redis.dev to Procfile.dev and updating the README so that it tells devs to have ES and Redis running before they run foreman.